awesome.md

```diff
@@ -0,0 +1,38 @@
+# The awesome manifesto
+
+If you want your list to be included on `awesome`, you should really just include awesome stuff.
+
+But **what is awesome?**
+
+## Only awesome is awesome.
+
+Research if the stuff you're including is actually awesome. Put only stuff on the list you or another contributor can personally recommend and rather leave stuff out than include too much.
+
+## Comment on why something is awesome
+
+Apart from suggesting a particular item on your list, you also should tell your readers _why_ you do suggest it and how the potential users will benefit from using it.
+
+## Make it clear on what the list is about
+
+Make sure your list covers a certain scope and nothing else.
+Link to other awesome lists if you think they already cover a certain subject good enough.
+
+## Pay attention to grammar
+
+Please be sure that you do your best to keep the list grammatically correct. Pay attention to Markdown formatting errors as well, only accept pull requests with proper formatting.
+
+## Choose an appropriate license
+
+See [Choose a License](http://choosealicense.com/no-license/) for more details. Keep in mind that if you haven't selected a license, it basically means the people are not allowed to reproduce, distribute or create derivative works.
+
+## Include contribution guidelines
+
+People who are contributing to your list should have a clear understanding how they should do so. You could copy a contributing.md from another awesome repository you like.
```

Comment:
This also needs to explain what the contributing guidelines should guide about: formatting of the list, order, ...